qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -929,21 +931,6 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                                 val_mapping={'ON': 1, 'OFF': 0},
```

Comment:
since `create_on_off_val_mapping` is introduced to this driver, shall it be applied to all the other parameters? :)

Reply:
probably not in this PR, this requires changes in the logic because the logic is wired to 'ON'/'OFF' values and should be rewired to use True/False.... next time)